### ISSUE TITLE
Predicting is a Tumor is Malignant or Benign

### ISSUE THREAD

**Issue by abhishek2828** [abhishek2828/Predicting-Tumour#1]
This project works on the basics of Deep Learning. We are given a dataset of Malignant and Benign Tumors. We are given different values for different characteristics. We use a neural network and with the help of logistic regression we train the network to predict the outcome for a given input data. 

In this project, I used the Python Programming Language and with the use of TensorFlow, MatPlot and Pandas, I could analyse the given dataset and predict the outcome of a tumor with close to 100% accuracy
